Design-Docs/glossary.md

```diff
@@ -0,0 +1,143 @@
+
+An Availability Zone allows OpenStack compute hosts to be divided into logical groups and provides a form of physical isolation and redundancy from other Availability Zones, for example by using a separate power supply or network devices.
+
+When users provision resources, they can specify in which Availability Zone their instance should be created. In this way, cloud customers can ensure that their application resources are distributed across different computers to achieve high availability in the event of a hardware failure.
+
+## Definition of Host Aggregates
+
+Host aggregates are a mechanism for partitioning compute nodes which is not explicitly visible to users in an OpenStack/SCS cloud.  Host aggregates are based on arbitrary characteristics such as server type, processor type, GPU, disk speed, etc.
+Administrators assign flavors to host aggregates by specifying metadata on the host aggregate and customizing the extra specifications of the flavor. It is then up to the Nova scheduler to determine the best match for the user request. Compute nodes can also be in more than one host aggregate.
+
+
+Optionally, one can designate a host aggregate as an Availability Zone, e.g. for simplificatin reasons of the user selection of an availbility zone.  
+Availability Zones differ from Host Aggregates in that they are shown to the user as a Nova boot option, so Compute VMs can be started on them.
+Compute Nodes, however, can only be in a single Availability Zone. We can configure a default Availability Zone where instances will be scheduled if the user does not specify an Availability Zone.
+
+Info: A prerequisite for creating an Availability Zone is a host aggregate.
+
+## Definition of a Cell
+
+The Cells paradigm simplifies the handling of large Openstack deployments. 
+
+Cells is an OpenStack Nova feature that improves scalability for Nova in OpenStack Platform. Each Cell has a separate database and message queue, which increases performance when scaling. One can provision additional Cells to handle large deployments, and compared to Regions, this allows access to a large number of compute nodes through a single API.
+
+Each Cell has its own Cell controllers running the database server and RabbitMQ along with the Nova Conductor services.
+
+Nova Conductor services, called "Super Conductor", continue to run on the main controller nodes.
+
+The services in the Cell Controllers can still call placement APIs, but cannot access other API layer services via RPC, nor can they access global API databases on the control nodes.
+
+## Definition of a Control Plane
+
+In Openstack/SCS, a Control Plane consists of 5 hardware nodes, which together serve several Availability Zones and thus provide a common usable API for a region. The Control Plane also shares the network (Neutron), the Sheduler and the CEPH services.
+
```

Comment:
That's quite optimistic to have all of that in 5 nodes btw.

Reply:
Mind game: all 5 CPs are in the same AZ, right? What happens to other AZ's if the the AZ with the CP burns down?